docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.10.0
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
+    environment:
+      CALYPSO_ENV: 'development'
+      NODE_ENV: 'development'
+      HOST: 'calypso.localhost'
+      PORT: 3000
+    ports:
+      - '3000:3000'
```

Comment:
I disagree wholeheartedly. That belongs to the infrastructure, systems, and firewalls. If I want it open to the world (to test on my mobile) it should just work. If I'm paranoid about port openings because I work on public wifi, then I'll likely have a firewall preventing it from being open to the network anyway.